end_to_end_tests/input/rest_api_with_authorizers/template.yaml

```diff
@@ -0,0 +1,134 @@
+AWSTemplateFormatVersion : '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Example SAM template with Cognito, Lambda token and Lambda request authorization for Rest Apis
+
+Resources:
+  MyApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      Auth:
+        Authorizers:
+          MyCognitoAuthorizer:
+            UserPoolArn:
+              Fn::GetAtt: MyCognitoUserPool.Arn
+          MyLambdaTokenAuth:
+            FunctionArn:
+              Fn::GetAtt: MyLambdaAuthFunction.Arn
+          MyLambdaRequestAuth:
+            FunctionPayloadType: REQUEST
+            FunctionArn:
+              Fn::GetAtt: MyLambdaAuthFunction.Arn
+            Identity:
+              QueryStrings:
+                - authorization
+
+  MyFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs12.x
+      InlineCode: |
```

Comment:
Should one of these tests not use inline so we can see how that is setup? I would assume that InlineCode is not going to be the majority of things we do with these tests.

Reply:
Good idea! I will update one of the tests without `InlineCode`

Reply:
Updated the `template.yaml` for `http_api_cors`integration test